mne/viz/tests/test_topomap.py

```diff
@@ -68,7 +68,7 @@ def test_plot_topomap():
 
     ev_bad = evoked.pick_types(meg=False, eeg=True, copy=True)
     ev_bad.pick_channels(ev_bad.ch_names[:2])
-    ev_bad.plot_topomap(times=ev_bad.times[:2] - 1e-6)  # auto, should plot EEG
+    ev_bad.plot_topomap(times=ev_bad.times[1:3] - 1e-6)  # auto, plots EEG
```

Comment:
The checker for topomap is now more strict. It doesn't allow any values outside the ``times`` window, so this test would fail. The checker had to change because ``_time_mask`` is more lenient now in allowing values to be anything, not just close to ``times``. In my opinion we could get rid of the checker for topomap altogether, but I think some people want it there.